config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   apipie
 
+  root to: 'users#new'
+  get "/auth/:provider/callback" => "sessions#create"
+  get "/signout" => "sessions#destroy", :as => :signout
```

Comment:
do not need `:as => signout`. All that is doing is creating a shortened url: `signout_path_` which is not being used anywhere (and shouldnt be used)